source/Tipster.cpp

```diff
@@ -57,27 +56,28 @@ Tipster::Tipster()
 	fURL = new BString("");
 	fArtworkTitle = new BString("");
 	fPreviousTip = new BString("");
+	fRunner = NULL;
 
 	fTipsterTextView = new TipsterText();
 	fIcon = new BButton("iconview", "", new BMessage(OPEN_URL));
+	fIcon->SetFlat(true);
```

Comment:
TBH, I like the un-flat look better, esp. for the Replicant. Having it look like a button also indicates that there's something to click.

Reply:
The first tip says to press the icon not the button...

Reply:
It may even have been a flat button before. In any case, talking about the "icon" is still OK, since icons are so very striking, and the fact that it's also a button is secondary.